### ISSUE TITLE
IESG evaluation record is not shown

### ISSUE THREAD

**Issue by smyslov** [ietf-tools/datatracker#5493]
### Describe the issue

When I-D is scheduled for IESG telechat, a grid with individual AD ballots is shown right to the document. Before, when you click on this grid box, a pop-up window showing IESG evaluation record appeared. Currently when you click on this grid box the screen just becomes darker and nothing more happens.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
I can't reproduce.
@smyslov - could you please verify you are still encountering this issue?
If so, could you look at the javascript console and forward anything it is complaining about?

**Comment by smyslov** [comment]
Yes, I stll encounter this issue. In particular, at https://datatracker.ietf.org/wg/ipsecme/documents/ when I click on IESG evaluation for draft draft-ietf-ipsecme-add-ike-13 (URL to be open is https://datatracker.ietf.org/doc/draft-ietf-ipsecme-add-ike/ballotpopup/915379/ , nothing happens (no popup window appears). When I open this URL in a new window (or a new tab) the content is shown as HTML page with no formatting. There is no any suspicious in JavaScript console (favicon.ico failed to load, but I don't think this matters). Two screenshots are attached.

I observe this behaviour on Edge (Chromium based) and Chrome (both desktop and mobile). Let me know what other information you need.

![ballot](https://user-images.githubusercontent.com/10504077/235607465-9fa5c822-96ed-4c60-827c-8c821decc215.jpg)
![ipsecme](https://user-images.githubusercontent.com/10504077/235607475-2b35237d-33db-4864-a1af-2aace11bd200.jpg)

**Comment by rjsparks** [comment]
Thank you - the pointer to exactly which page the issue is occurring on (https://datatracker.ietf.org/wg/ipsecme/documents/) is enough. I can replicate the issue now (I had been looking at /iesg/agenda and /iesg/agenda/documents).
